installer/Templates/Requirements.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+
+namespace Installer\Templates;
+
+
+use Template;
+
+class Requirements implements Template
+{
+
+	public function print(array $input = [])
+	{
+		echo "<br>";
+		foreach (
+			$input['requirements'] as $type => $requirement
+		) {
+			echo "<br>";
+			echo "\t".'<ul class="list">';
+			echo "\t"."\t".'<li class="list__item list__title ';
+			echo $input['phpSupportInfo']['supported'] ? 'success' : 'error';
```

Comment:
So this loop will run for `php` and `apache`? Why should `phpSupportInfo` be used for anything in case of the `apache` iteration?

Reply:
It does not matter in the case of Apache as it does not have a text at the right (that is the thing colored). The coloring of `<strong>` is what matters.